lib/perl/Genome/Sys/Lock.pm

```diff
@@ -208,7 +208,11 @@ sub all_backends {
 }
 
 sub scopes {
-    return ('site', 'tgisan', 'unknown');
+    if ($ENV{GENOME_SYS_ID} && $ENV{GENOME_SYS_ID} ne 'GMS1') {
+        return ('site', 'unknown');
+    } else {
+        return ('site', 'tgisan', 'unknown');
+    }
 }
```

Comment:
Malachi suggested we let the scopes be in the environment entirely for the standalone version.

This would let the internal version switch to env too instead of having hard-coded values:
```
sub scopes {
   if ($ENV{GENOME_LOCK_SCOPES}) {
      return split(":", $ENV{GENOME_LOCK_SCOPES})
   }
   else {
      # Ultimately we should move this into the TGI config and not have it in code at all.
      return ('site', 'tgisan', 'unknown');
   }
}
```